src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Formatting/FormattingContext.cs

```diff
@@ -255,8 +255,8 @@ private static void ValidateComponents(RazorCodeDocument oldCodeDocument, RazorC
             RazorCodeDocument codeDocument,
             FormattingOptions options,
             AdhocWorkspaceFactory workspaceFactory,
-            bool isFormatOnType = false,
-            bool automaticallyAddUsings = false)
+            bool isFormatOnType,
+            bool automaticallyAddUsings)
```

Comment:
I could imagine us one day pulling the "add using" portion out into a separate mechanism. Just food for thought